sonar-python-plugin/src/main/java/org/sonar/plugins/python/TextReportReader.java

```diff
@@ -31,11 +31,19 @@
 import org.sonar.api.utils.log.Logger;
 import org.sonar.api.utils.log.Loggers;
 
-public class Flake8ReportReader {
+/**
+ * Common implementation to parse Flake8 and Pylint reports
+ */
+public class TextReportReader {
+
+  private final int reportOffset;
```

Comment:
_nitpick_ I would add an empty line here just to indicate that this is an instance field. Maybe also would move it below the static fields.